go.mod

```diff
@@ -64,6 +64,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/urfave/cli v1.22.4
 	github.com/vmware/govmomi v0.22.2
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200716221620-18dfb9cca345
```

Comment:
```suggestion
```
there is some error, it should be with /v3, like in the line below

Reply:
Unfortunately I don't think it's an error. it's a dependency introduced by /v3 itself. /v3 doesn't expose an interface to work with types like `member`, instead it just returns the internal type: https://github.com/kubermatic/kubermatic/blob/bef9d98285371cd8415a24ec7464e775e7f4da27/cmd/etcd-launcher/main.go#L355, I couldn't find a simple workaround for this.

Reply:
@kron4eg fixed, thanks man!